index.js

```diff
@@ -79,18 +79,45 @@ app.post('/signin',(req,res)=>{
 })
 app.put('/update',(req,res)=>{
     //update the username
+    const newUsername = req.body.name;
```

Comment:
why checking with name can do with email instead as email is unique in each user entry , name can have duplicates